src/main/java/io/kestra/plugin/ai/provider/DashScope.java

```diff
@@ -0,0 +1,159 @@
+import dev.langchain4j.model.chat.ChatModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.model.image.ImageModel;
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.ai.domain.ChatConfiguration;
+import io.kestra.plugin.ai.domain.ModelProvider;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+
+import java.time.ZoneId;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonDeserialize
+@Schema(title = "DashScope (Qwen) Model Provider")
+@Plugin(
+    examples = {
+        @Example(
+            title = "Chat completion with DashScope (Qwen)",
+            full = true,
+            code = {
+                """
+                    id: chat_completion
+                    namespace: company.ai
+
+                    inputs:
+                      - id: prompt
+                        type: STRING
+
+                    tasks:
+                      - id: chat_completion
+                        type: io.kestra.plugin.ai.ChatCompletion
+                        provider:
+                          type: io.kestra.plugin.ai.provider.DashScope
+                          apiKey: "{{ kv('DASHSCOPE_API_KEY') }}"
+                          modelName: qwen-plus
+                        messages:
+                          - type: SYSTEM
+                            content: You are a helpful assistant, answer concisely, avoid overly casual language or unnecessary verbosity.
+                          - type: USER
+                            content: "{{ inputs.prompt }}"
+                    """
+            })
+    },
+    aliases = "io.kestra.plugin.langchain4j.provider.DashScope")
+public class DashScope extends ModelProvider {
+    private static final String DASHSCOPE_CN_URL = "https://dashscope.aliyuncs.com/api/v1";
+    private static final String DASHSCOPE_INTL_URL = "https://dashscope-intl.aliyuncs.com/api/v1";
+    private static final String DASHSCOPE_BASE_URL =
+        ZoneId.systemDefault().equals(ZoneId.of("Asia/Shanghai"))
+            ? DASHSCOPE_CN_URL
+            : DASHSCOPE_INTL_URL;
+    @Schema(title = "API Key")
+    @NotNull
+    private Property<String> apiKey;
+
+    @Schema(
+        title = "API base URL",
+        description =
+            """
+                  If you use a model in the China (Beijing) region, you need to replace the URL with: https://dashscope.aliyuncs.com/api/v1,
+                  Otherwise use the Singapore region of: "https://dashscope-intl.aliyuncs.com/api/v1
+              """)
```

Comment:
The comment is misleading as the task will guess the default URL based on the system timezone.
Can you explain that in the comment?